api/src/main/resources/marquez/db/migration/V66.1__job_facets_changes.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE job_facets ALTER COLUMN lineage_event_type DROP NOT NULL;
+ALTER TABLE job_facets DROP CONSTRAINT job_facets_run_uuid_fkey;
+ALTER TABLE job_facets ADD COLUMN job_version_uuid uuid REFERENCES job_versions (uuid);
+
+CREATE INDEX job_facets_job_version_uuid ON job_facets (job_version_uuid);
+
+ALTER TABLE lineage_events ADD COLUMN spec_event_type VARCHAR(64);
```

Comment:
I would use `_event_type` to signify that it's a "private" column that we define external from the OL spec.

Reply:
renamed column to `event_type`